### PR TITLE
Add Question 1 Page card (Fixes #40)

### LINKED ISSUE #40: Question 1 Page: One card (+ interaction)
image

### PR DESCRIPTION
**Pull request by jshiii117**
#### What is the purpose of your change?
- Adding new component

Fixes #40 

#### What areas of the application does this change impact? (changes API, folder structure, etc)
- Changes frontend

#### What features/fixes are deferred/not handled in this PR?
- Does not include parent container for holding individual cards

## Checklist
- [ ] Adds unit tests
- [X] Has a work item linked to the PR

### CODE REVIEW

File: tmt-app/src/Components/QuestionCard.js
Comment (jwu910) [inline tmt-app/src/Components/QuestionCard.js:1]: can we use `jsx` for extensions for files with jsx in it? helps distinguish between javascript files and files with react jsx code. Ideally we can use util files or helper files with `.js` endings to indicate its mostly just plain javascript in that file, `.jsx` makes it easier to tell that this file is a component

File: tmt-app/src/Components/QuestionCard.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+      ? "white"
+      : CustomTheme.palette.BobaHighlight.main,
+    color: isHovered ? "black" : "white",
+  };
+
+  return (
+    <Grid
+      item
+      key={item.title}
+      style={style}
```
Comment (jwu910) [inline tmt-app/src/Components/QuestionCard.js:55]: we should consider using the `sx` prop over `style` when possible https://mui.com/system/getting-started/the-sx-prop/

File: tmt-app/src/Components/QuestionCard.js [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+      onClick={handleClick}
+      onMouseOver={handleHover}
+      onMouseLeave={handleLeave}
+    >
+      {isSelected ? (
+        <>
```
Comment (jwu910) [inline tmt-app/src/Components/QuestionCard.js:61]: The Grid parent item may or may not be helpful. we had mentioned over the call that the grid might be too rigid for something like this as working with flex and auto wrapping elements at the responsibility of the browser would probably be less work for us.

Lets take that into consideration, if the parent is no longer Grid, do we want a Fragment here, or do we want another MUI element we could add some styles to?

**Review by jwu910 — CHANGES_REQUESTED** [review]
left some comments, will like to see some changes made.

If you can also provide some screenshots or gifs of the component as it is, that would be helpful to preview the work, looking forward to the next review

File: tmt-app/src/Components/QuestionCard.js [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+    alignItems: "center",
+    width: "240px",
+    minHeight: "152px",
+    boxShadow: "6px 6px 16px rgba(0, 0, 0, 0.16)",
+    borderRadius: "36px",
+    flex: "none",
+    flexGrow: 0,
+    textAlign: "center",
+    transition: "all 0.3s ease-in-out",
+    border: isSelected
```
[the remark this answers is not in the document]
Comment (jshiii117) [reply tmt-app/src/Components/QuestionCard.js:38]: Thanks for reviewing the changes Josh! Just a bit confused by this one, what is meant by sorting the objects?

File: tmt-app/src/Components/QuestionCard.jsx [hunk 4]
```diff
@@ -19,45 +18,41 @@ const QuestionCard = ({ item }) => {
...
     }
   }
 
   return (
-    <Grid
-      item
+    <Box
       key={item.title}
-      style={style}
+      style={sx}
```
Comment (jwu910) [inline tmt-app/src/Components/QuestionCard.jsx:55]: Fyi, using a variable named `sx` won't be the same as using the `sx` prop available through MUI components https://mui.com/system/getting-started/the-sx-prop/

**Review by michachan — APPROVED** [review]
Awesome, looks good to me!

**Review by jwu910 — APPROVED** [review]

**Pull request merged**